### PR TITLE
WD_LOGIN PAGE_ABANOB MIKHAEL

### PR DESCRIPTION
**Pull request by ABANOBMIKHAEL**
Fixed
1. The  the color palette
2. The text box size
3. navbar

### CODE REVIEW

File: next_webapp/public/styles/licensing.css
Comment (adrianweerasinghe97) [inline next_webapp/public/styles/licensing.css:1]: looks good

File: next_webapp/public/styles/login.css
Comment (adrianweerasinghe97) [inline next_webapp/public/styles/login.css:1]: I would recommend getting rid of the comments following the css. you could add the comment above the css if necessary

File: next_webapp/src/app/login/page.tsx
Comment (adrianweerasinghe97) [inline next_webapp/src/app/login/page.tsx:1]: This code seems to have several issues, including duplicate declarations, inconsistent code structures, and nested blocks that should not be nested.

**Review by adrianweerasinghe97 — COMMENTED** [review]
1. The licensing.css looks good
2. For the login.css you  could get rid of the comments following the css. you could add the comment above the css if necessary
3. For the page.tsx, the code seems to have several issues, including duplicate declarations, inconsistent code structures, and nested blocks that should not be nested.

**Comment by ABANOBMIKHAEL** [comment]
CSS, I removed the comments.
Login(Page.tsx)
1.Removed Duplicate Declarations
2.Consistent Code Structure
Please let me know if there is anything else.
Thank you

**Comment by kaseyuu** [comment]
Hi Abanob, could you please resolve the conflicts?

**Review by kaseyuu — APPROVED** [review]

**Pull request merged**